Makefile

```
@@ -86,24 +86,17 @@ create-data-harmonizer:
 
 # Note: `all` is an alias for `site`.
 all: site
-
-# TODO: Document this make target.
-site: clean site-clean gen-project gendoc nmdc_schema/gold-to-mixs.sssom.tsv accepting-legacy-ids-all
-
-# may change files in nmdc_schema/ or project/. uncommitted changes are not tolerated by mkd-gh-deploy
+site: clean site-clean gen-project gendoc \
+nmdc_schema/gold-to-mixs.sssom.tsv \
+nmdc_schema/nmdc_materialized_patterns.schema.json nmdc_schema/nmdc_materialized_patterns.yaml \
+migration-doctests
```

I added some indentation and put each filename on its own line (rather than 1 on one line, 2 on the next line [even though they are similar], 1 on the next line).

```suggestion
site: clean site-clean gen-project gendoc \
      nmdc_schema/gold-to-mixs.sssom.tsv \
      nmdc_schema/nmdc_materialized_patterns.schema.json \
      nmdc_schema/nmdc_materialized_patterns.yaml \
      migration-doctests
```

---

It just occurred to me that lines like this often precede tab-indented lists of commands; in which case, this new indentation (I used spaces) might make the file more difficult to read instead of less difficult.

---

I agree that this is easier to read but I am hesitant to add white-space in a Makefile (which is already picky about XXX vs tabs). If we did that, I would want to apply it to all cases where a list of target prerequisites spills over the line on which the target is defined.

---

Thanks for addressing this topic in conversation here. I'll resolve this conversation. I'm OK with proceeding with this snippet as it exists today.